pkgs/by-name/op/opengfw/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "opengfw";
+  version = "0.4.0";
+
+  CGO_ENABLED = 0;
+  vendorHash = "sha256-F8jTvgxOhOGVtl6B8u0xAIvjNwVjBtvAhApzjIgykpY=";
+  src = fetchFromGitHub {
+    owner = "apernet";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-kmbG6l5CtZGM/zpvl2pukq5xsOIy28RDyb4sHBsoyOw=";
+  };
+
+  meta = with lib; {
```

Comment:
could you avoid using `with lib` here? because it is a problematic anti-pattern, refer to: https://github.com/NixOS/nixpkgs/issues/208242